### PR TITLE
Remove Latin phrases from the docs

### LINKED ISSUE #62224: Remove Latin phrases from the docs
##### SUMMARY
Latin words and phrases like `e.g.` or `etc.` are easily understood by native English speakers and speakers of European languages who speak some English. They are harder to understand for speakers of Asian languages who speak some English. They are also tricky for automated translation.

Add a row to the Style Guide cheat-sheet about avoiding Latin words and phrases.
Remove existing Latin words and phrases from the documentation.

##### ISSUE TYPE
- Documentation Report

##### COMPONENT NAME
docs.ansible.com

##### ANSIBLE VERSION
all

##### CONFIGURATION
N/A

##### OS / ENVIRONMENT
N/A

### PR DESCRIPTION
**Pull request by mcbenjemaa**
##### SUMMARY

add styleguide about avoiding use of latin words"

TODO: replace latin words

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

docs.ansible.com

##### ADDITIONAL INFORMATION

```paste below
IN_PROGRESS
```

Fixes #62224

### CODE REVIEW

**Comment by medmedchiheb** [comment]
component docs

**Comment by medmedchiheb** [comment]
docs.ansible.com

**Comment by medmedchiheb** [comment]
cc @acozine @gundalow
I need first review for my current changes!
i have added a row to styleguide and started to replace latin words in the community section docs.
Shall i continue!

File: docs/docsite/rst/dev_guide/style_guide/index.rst [hunk 1]
```diff
@@ -48,6 +48,30 @@ Header case
 Headers should be written in sentence case. For example, this section's title is
 ``Header case``, not ``Header Case`` or ``HEADER CASE``.
 
+
+Avoid using latin phrases
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/style_guide/index.rst]:
```suggestion
Avoid using Latin phrases
```

File: docs/docsite/rst/dev_guide/style_guide/index.rst [hunk 2]
```diff
@@ -48,6 +48,30 @@ Header case
...
+-------------------------
+
+Latin words and phrases like ``e.g.`` or ``etc.``
+are easily understood by native English speakers and some European who speak English.
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/style_guide/index.rst]:
```suggestion
are easily understood by English speakers.
```

File: docs/docsite/rst/dev_guide/style_guide/index.rst [hunk 3]
```diff
@@ -48,6 +48,30 @@ Header case
...
+They are harder to understand for speakers of Asian languages who speak some English. They are also tricky for automated translation..
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/style_guide/index.rst]:
```suggestion
They may be harder to understand for others and are also tricky for automated translation.
```

File: docs/docsite/rst/dev_guide/style_guide/index.rst [hunk 4]
```diff
@@ -48,6 +48,30 @@ Header case
...
+
+Latin words/phrases:
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/style_guide/index.rst]:
```suggestion
Use the following English terms in place of Latin terms or abbreviations: 
```

**Review by samccann — COMMENTED** [review]
Thanks for kickstarting this!

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @medmedchiheb for taking on this Ansible docs issue and PR!

**Pull request merged**